src/components/calculator.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './Calculator.css';
+
```

Comment:
## Your file name
![css](https://user-images.githubusercontent.com/93039359/184025587-e1ecaa5d-6548-43a3-b541-d7f5bdeed6e9.png)

- [ ] You can see that your file has a small letter at the start of its name and you are importing this file with the first letter capital. Please import it with the first letter small so you won't face any error :+1: